.github/workflows/regressions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import datetime
+import os
+import subprocess
+
+dir_path = os.path.dirname(os.path.abspath(__file__))
+regressions_dir = os.path.join(dir_path, 'regression_yamls')
+
+from mcli import RunConfig, create_run
+
+
+def get_configs(cluster: str, mpt_7b_ckpt_path: str, wandb_entity: str,
+                wandb_project: str):
+    eval_7b_hf = RunConfig.from_file(
+        os.path.join(regressions_dir, 'eval-7b-hf.yaml'))
+    eval_7b_composer = RunConfig.from_file(
+        os.path.join(regressions_dir, 'eval-7b-composer.yaml'))
+    llama2_finetune = RunConfig.from_file(
+        os.path.join(regressions_dir, 'llama2-finetune.yaml'))
+    mpt_125m_chinchilla = RunConfig.from_file(
+        os.path.join(regressions_dir, 'mpt-125m-chinchilla.yaml'))
+    mpt_125m_sharded_resumption = RunConfig.from_file(
+        os.path.join(regressions_dir, 'mpt-125m-sharded-resumption.yaml'))
+
+    # make specific changes
+    eval_7b_composer.parameters['models'][0]['load_path'] = mpt_7b_ckpt_path
+
+    all_configs = [
+        eval_7b_hf, eval_7b_composer, llama2_finetune, mpt_125m_chinchilla,
+        mpt_125m_sharded_resumption
+    ]
+
+    commit_hash = subprocess.check_output(['git', 'rev-parse',
+                                           'HEAD']).strip().decode('utf-8')
+    timestamp = datetime.datetime.now().strftime('%m-%d-%Y::%H:%M:%S')
+    wandb_group = f'{timestamp}::{commit_hash}'
+
+    # make general changes
+    loggers = {
+        'wandb': {
+            'entity': wandb_entity,
```

Comment:
this will be hardcoded to a specific user for future regression tests but is fine for now :)